src/include/quick-lint-js/error.h

```diff
@@ -142,6 +142,14 @@
       error_unexpected_identifier, { source_code_span where; },                \
       .error(u8"unexpected identifier", where))                                \
                                                                                \
+  QLJS_ERROR_TYPE(                                                             \
+      error_expected_expression, { source_code_span where; },                  \
```

Comment:
Nit: Order this alphabetically in the error list.

Reply:
I just pushed an updated version with the fixes.